docs/administrators/logging.rst

```diff
@@ -86,59 +63,98 @@ file can be found at ``/usr/lib/systemd/system/inmanta-server.service``.
   Type=simple
   User=inmanta
   Group=inmanta
-  ExecStart=/usr/bin/inmanta --log-file /var/log/inmanta/server.log --log-file-level 2 --timed-logs server
+  ExecStart=/usr/bin/inmanta --log-file /var/log/inmanta/server.log --log-file-level INFO --timed-logs server
   Restart=on-failure
 
   [Install]
   WantedBy=multi-user.target
 
-The ``--log-file-level`` takes the log-level as an integer, where ``0=ERROR``, ``1=WARNING``, ``2=INFO`` and ``3=DEBUG``.
-
-Apply the changes by reloading the service file and restarting the Inmanta server:
 
 .. code-block:: sh
 
   sudo systemctl daemon-reload
   sudo systemctl restart inmanta-server
 
 
-Log level manually started agent
---------------------------------
+Fine grained configuration
+----------------------------
+For fine grained configuration, `a standard python dict config file <https://docs.python.org/3/library/logging.config.html#logging-config-dictschema>`_ can be passed in via the config file for each component individually:
+
+.. code-block:: yaml
+
+    [logging]
+    server = server_log.yml
+    scheduler = scheduler.log.tmpl
+    compiler = compiler.yml
+
+or via a cli option:
+
+.. code-block:: sh
+
+    inmanta --logging-config server_log.yml server
+
+
+The log config has to be either a ``yaml`` file, containing a `python dict config <https://docs.python.org/3/library/logging.config.html#logging-config-dictschema>`_ or a template of a ``yaml`` file. In this case, the file name has to end with ``tmpl``.
```

Comment:
Any plan to also support the `.yml` extension?

Reply:
if it is not a template, it can be whatever

Reply:
The default configs in this document are `.yml` so I would expect both extensions to be valid (and we should document it)

Reply:
(Just saw your comment the template is also `.yml`)